troubleshooting/uefi-troubleshooting.md

```diff
@@ -7,6 +7,19 @@ permalink: /doc/uefi-troubleshooting/
 Troubleshooting UEFI related problems
 ========================================
 
+Change installer kernel parameters in UEFI
+---------------------
+
+If you've installed successfully in legacy mode but had to change some kernel parameters for it to work, you should try installing in UEFI with the same kernel parameters.
+
+Change the `xen.cfg` on a USB media
+
+01. Attach the usb disk, find the EFI partition and mount it
```

Comment:
I'd add a hint it is the second partition. Or the one with just "EFI" directory, not all other stuff like "Packages", "isolinux" etc. Generally give some info how to "find the EFI partition".